book/website/ethical-research/data-anonymization.md

```diff
@@ -0,0 +1,129 @@
+
+# Data Anonymization
+
+## Prerequisites / recommended skill level
+The following sections of the Reproducible Research book will better help understand the motivation for this chapter:
+<!--would appreciate some help with linking the sections to this table-->
+| Prerequisite | Importance |
+| -------------|----------|
+| {ref}`rr-open`| Helpful |
```

Comment:
Perhaps this page on sharing and archiving data: https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html?highlight=anonym
(and you probably want to update that page, which briefly mentions anonymisation, to link to this one!)